jp/components/grid_toolbar.md

```diff
@@ -0,0 +1,174 @@
+﻿---
+title: グリッド ツールバー コンポーネント - ネイティブ Angular | Ignite UI for Angular
+_description: Ignite UI for Angular Grid Toolbar コンポーネントは、IgxGrid 機能 UI コントロールを一元管理できます。
```

Comment:
Can we use ツールバー for Toolbar?  Since it's not a control name, but a feature?

Reply:
@yoshimis If you could update the metadata for the description and the keywords to translate "Toolbar", I think that would help with searching.  Otherwise it's not critical, it's ok as-is.

Reply:
done. https://github.com/IgniteUI/igniteui-docfx/compare/YS_0627?expand=1